Makefile

```diff
@@ -292,8 +292,13 @@ deploy-onprem:
 deploy-onprem-for-subsystem:
 	export DUMMY_IGNITION="true" && $(MAKE) deploy-onprem
 
+setup-openshift-ci:
+	setup_prow_ci_env.sh
```

Comment:
There's already `make build-openshift-ci-test-bin` target being called on https://github.com/openshift/release/blob/357ca3f0f913a28534924305b31aa66aefdbd016/ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml#L28-L29
You should use it.